### PR TITLE
feat(site): Add MVP implementation

### PR DESCRIPTION
**Pull request by jkuester**
I believe what we have now is a good "minimum viable product."  It is usable and stable. So this seems like a good time to go to `main` and to start doing closer reviews on subsequent changes.

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: GitHub pages
+
+on:
+  push:
+    branches:
+      - main
+      - initial-dev
```
Comment (jkuester) [inline .github/workflows/deploy.yaml:7]: Should we remove this now or wait until the PR is merged?

**Comment by TobiTenno** [comment]
Let's remove that now before it's in main and we forgotten bloat

**Comment by jkuester** [comment]
Whoops, just realized that we still have a couple of open issues for the MVP project: https://github.com/orgs/LibreWeather/projects/1

https://github.com/LibreWeather/libre-weather/issues/3
https://github.com/LibreWeather/libre-weather/issues/2

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (TobiTenno) [reply .github/workflows/deploy.yaml:7]: Let's remove now

File: src/assets/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (TobiTenno) [inline src/assets/manifest.json:3]: Let's update this so it's specific to LibreWeather

```suggestion
  "short_name": "LibreWeather",
  "name": "LibreWeather Frontend",
```

File: src/assets/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
```
Comment (TobiTenno) [inline src/assets/manifest.json:23]: update theme color to match the blue in libreweather's logo?

File: src/components/DailyOverview.jsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      condition,
+      temp: tempDisplay(temp),
+      time,
+    }))
+  ),
+});
+
+const getConditionClass = (condition) => {
+  switch (condition) {
+    // TODO Should we support light rain?
```
Comment (TobiTenno) [inline src/components/DailyOverview.jsx:23]: Log an issue to discuss support for light rain, reference this line, and remove TODO

File: src/components/DailyOverview.jsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+      return 'mostlyCloudy';
+    default:
+      return 'clear';
+  }
+};
+
+const getConditionLabel = (condition, hrCnt, time) => {
+  let label = '';
+  switch (condition) {
+    // TODO Should we support light rain?
```
Comment (TobiTenno) [inline src/components/DailyOverview.jsx:41]: as above

File: src/components/WeeklyForecast.jsx [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+  render() {
+    const { dailyWeather, index, overallMinTemp, overallMaxTemp } = this.props;
+    const { drawerDisplay, drawerIcon } = this.state;
+
+    const conditionDate = new Date();
+    conditionDate.setHours(12);
+    const conditionTime = conditionDate.getTime()/1000;
+    const {precipitation} = dailyWeather;
+    
+    // TODO the tempature range details need to be updated to include the times (calculated by looking at the hourly data)
```
Comment (TobiTenno) [inline src/components/WeeklyForecast.jsx:134]: Log issue, remove TODO

File: src/index.jsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (TobiTenno) [inline src/index.jsx:23]: log issue to set up PWA

File: src/tests/App.test.jsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (TobiTenno) [inline src/tests/App.test.jsx:9]: i'm guessing this doesn't really work, so we may want to look at replacing it with a different basic element test

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (TobiTenno) [reply .github/workflows/deploy.yaml:7]:
```suggestion
```

File: src/components/DailyOverview.jsx (hunk 4 above)
Comment (TobiTenno) [reply src/components/DailyOverview.jsx:23]: logged #9

File: src/components/DailyOverview.jsx (hunk 5 above)
Comment (TobiTenno) [reply src/components/DailyOverview.jsx:41]: logged #9

File: src/components/WeeklyForecast.jsx (hunk 6 above)
Comment (TobiTenno) [reply src/components/WeeklyForecast.jsx:134]: Logged #8

File: src/index.jsx (hunk 7 above)
Comment (TobiTenno) [reply src/index.jsx:23]: Logged #7

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**